### PR TITLE
update zowe Desktop docs

### PR DESCRIPTION
**Pull request by Himani1519**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

**Review by lparadkar-rocket — APPROVED** [review]

**Review by lparadkar-rocket — APPROVED** [review]

File: docs/user-guide/mvd-using.md [hunk 1]
```diff
@@ -48,6 +50,23 @@ To log out, click the User icon in the lower right corner and click **Sign Out**
...
+2. Locate the application you want to open in new tab.
+3. Right-click the application icon and select **Open In New Browser Tab**.
+
+  - While opening an application in new tab you can also do the following:
+   
+     -  You can use url to send data to the application, for example you would specify
+        `https://<gwshostname>:<gwsport>//ZLUX/plugins/org.zowe.zlux.bootstrap/web/?pluginId=org.zowe.editor:data:{"type":"openFile","name":"<path of file>"}`
+     -  You can use url to open application directly on browser with and without credentials using
+         `showLogin` in url.
+            1. If `showLogin = true` then you need to login with your credntials before using an
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:65]: *credentials

File: docs/user-guide/mvd-using.md [hunk 2]
```diff
@@ -74,35 +93,29 @@ Use the Languages setting in the Preferences panel to change the desktop languag
 
 ## Zowe Desktop application plug-ins
 
-Application plug-ins are applications that you can use to access the mainframe and to perform various tasks. Developers can create application plug-ins using a sample application as a guide. The following application plug-ins are installed by default:
+Application plugins are applications that you can use to access the mainframe and to perform various tasks. Developers can create application plugins using a sample application as a guide. The following application plug-ins are installed by default:
 
 ### Hello World Sample
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:98]: plugin --> plugins

You should mention that we currently have Sample plugins for Iframes, React, and Angular (each with their own Git link)
sample-iframe-app, sample-angular-app, sample-react-app etc

File: docs/user-guide/mvd-using.md [hunk 3]
```diff
@@ -74,35 +93,29 @@ Use the Languages setting in the Preferences panel to change the desktop languag
...
+The VT Terminal plugin provides a user interface that emulates the basic functions of DEC VT family terminals. On the "back end," the plugin and the Zowe Application Server connect to VT compatible hosts, such as z/OS UNIX System Services (USS), using standard network protocols.
 
 ### API Catalog
-The API Catalog plug-in lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
+The API Catalog plugin lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
 
 ### Editor
 With the Zowe Editor you can create and edit files and view datasets on the system that Zowe serves.
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:111]: 1. You can create, edit, and modify the properties of files such as ownership and permissions. You can also view and edit datasets. Navigation is controlled by the File Tree (link to zlux-file-explorer Git repo). Other actions are available using the top left menu, a toolbar above the tree, or hotkeys.

File: docs/user-guide/mvd-using.md [hunk 4]
```diff
@@ -4,17 +4,18 @@ You can use the Zowe&trade; Application Framework to create application plug-ins
...
+From a supported browser, open the Zowe Desktop at `https://zowe.externalDomains[0]:zowe.externalPort/zlux/ui/v1/` or you can navigate to the direct Desktop URI at `https://zowe.externalDomains[0]:zowe.externalPort/zlux/ui/v1/ZLUX/plugins/org.zowe.zlux.bootstrap/web/index.html`
     
 Where:
 
--   *myHost* is the host on which you are running the Zowe Application Server.
--   *httpsPort* is the value that was assigned to *node.https.port* in `server.json`.
-    For example, if you run the Zowe Application Server on host *myhost* and the value that is assigned to *node.https.port* in `server.json` is 12345, you would specify `https://myhost:12345/ZLUX/plugins/org.zowe.zlux.bootstrap/web/index.html`.
+-   *zowe.externalDomains[0]* is the host on which you are running the Zowe Application Server, its the 
+    value that was assigned in the zowe configuration file.
+-   *zowe.externalPort*  is the value that was assigned in the zowe configuration file.
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:17]: *zowe.externalPort* is the value of Gateway port that was assigned in the zowe configuration file.

File: docs/user-guide/mvd-using.md [hunk 5]
```diff
@@ -4,17 +4,18 @@ You can use the Zowe&trade; Application Framework to create application plug-ins
...
+    For example, if you run the Zowe Application Server on host *zowe.externalDomains[0]* and the value that is assigned to *zowe.externalPort* in the zowe configuration file is 12345, you would specify `https://zowe.externalDomains[0]:12345/zlux/ui/v1/ZLUX/plugins/org.zowe.zlux.bootstrap/web/index.html`.
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:18]: You can remove line 18 entirely. It's made redundant by lines 11-17 we don't need to be annoying

File: docs/user-guide/mvd-using.md [hunk 6]
```diff
@@ -74,35 +92,41 @@ Use the Languages setting in the Preferences panel to change the desktop languag
 
 ## Zowe Desktop application plug-ins
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:93]: plug-ins --> plugins (do a search all and replace for this file)

File: docs/user-guide/mvd-using.md [hunk 7]
```diff
@@ -74,35 +92,41 @@ Use the Languages setting in the Preferences panel to change the desktop languag
...
 ### API Catalog
-The API Catalog plug-in lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
+The API Catalog plugin lets you view API services that have been discovered by the API Mediation Layer. For more information about the API Mediation Layer, Discovery Service, and API Catalog, see [API Mediation Layer Overview](../getting-started/overview.md).
 
 ### Editor
-With the Zowe Editor you can create and edit files and view datasets on the system that Zowe serves.
-
-### Workflows
-From the Workflows application plug-in you can create, manage, and use z/OSMF workflows to manage your system.
+With the Zowe Editor you can create, edit, and modify the properties of files such as ownership and permissions. You can also view and edit datasets. Navigation is controlled by the File Tree https://github.com/zowe/zlux-file-explorer. Other actions are available using the top left menu, a toolbar above the tree, or hotkeys.
```
Comment (DivergentEuropeans) [inline docs/user-guide/mvd-using.md:122]: `[File Tree](https://github.com/zowe/zlux-file-explorer)`

You could change it to something like that for a cleaner hyperlink

**Review by DivergentEuropeans — APPROVED** [review]

**Comment by DivergentEuropeans** [comment]
Ready to merge ✅

**Pull request merged**